src/plots/cartesian/dragbox.js

```diff
@@ -543,8 +548,12 @@ module.exports = function dragBox(gd, plotinfo, x, y, w, h, ns, ew) {
             pushActiveAxIds(yaLinked);
         }
 
+        updates = {};
         for(i = 0; i < activeAxIds.length; i++) {
-            doTicks(gd, activeAxIds[i], true);
+            var axId = activeAxIds[i];
+            doTicks(gd, axId, true);
+            var ax = getFromId(gd, axId);
+            updates[ax._name + '.range'] = ax.range.slice();
```

Comment:
> so I think I should change it back to the previous form.

I agree. Listening to `plotly_relayout` and inspecting the event data for `xaxis.range[0]` or `xaxis.range[1]` must be pretty common in userland. We should try to clean that up in v2 or maybe we could add a `plotly_rerange`?